tests/regression/tests/REQUEST-932-APPLICATION-ATTACK-RCE/932101.yaml

```diff
@@ -24,5 +24,4 @@ tests:
             data: "arg=ifconfig;.+rm+-rf+/;+Something+%26%238222%3BThe+Title%26%238221%3B.+After+space+or+new+line+more+characters"
             version: HTTP/1.0
           output:
```

Comment:
This is the only place where you've removed the `# status: 403` comment. Is that on purpose?

Reply:
Yes, because it is not being used. Now the check goes to find the id in the logs, so it is not failing with 403.

Reply:
Hm. I don't follow. The other tests also use `log_contains`, so why didn't you remove the comments from the other rules?

Reply:
Oh, gotcha now! Well... I think the best is to remove them, as you said. In any case, there is the git history.